src/Castle.VisualStudio.NVelocityLanguageService/Castle.VisualStudio.NVelocityLanguageService-vs2013.csproj

```diff
@@ -57,7 +57,10 @@
   <ItemGroup>
     <Reference Include="Microsoft.VisualStudio.OLE.Interop" />
     <Reference Include="Microsoft.VisualStudio.Package.LanguageService.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
-    <Reference Include="Microsoft.VisualStudio.Shell.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />

```

Comment:
Is there a reason the VS2013 file is changing to reference the VS2015 assemblies?